circuits-circom/utils/shift_and_pack_venmo_payee_id.circom

```diff
@@ -0,0 +1,59 @@
+pragma circom 2.1.5;
+
+include "@zk-email/circuits/helpers/extract.circom";
+
+// Adapted from ShiftAndPack function in zkemail to shift, remove line break, and pack.
+// https://github.com/zkemail/zk-email-verify/blob/main/packages/circuits/helpers/extract.circom
+template ShiftAndPackVenmoPayeeId(in_array_len, max_substr_len, pack_size) {
+    var max_substr_len_packed = ((max_substr_len - 1) \ pack_size + 1);
+
+    // Include unrevealed 3 0s at the end, which we will remove prior to packing
+    // e.g. [67,64,65,65,48,61,13,10,48,44] => [67,64,65,65,48,61,13,10,48,44,0,0,0]
```

Comment:
This holds correct if `ceil(max_len_registration_id/pack_size)` in registration circuit is same as `ceil(max_len_payee_id/pack_size)` then there would be no extra zeroes in the packed output due to the extra 3 zeroes in the registration circuit. If they are the same, then the output array len of both these computations would be the same leading to the same hash. If not the, it will be different.

Reply:
To get the same hash, the input to the hashes has to be the same. 
The input to the hashes are output of packed bytes.
So the output of packed bytes (which is an array) should be the same. 
Output of the packed bytes would be the same, because we are only suffixing zeroes. But the output array len could be different. So to ensure the output array len is the same, we need to make sure the `max_len` is the same.
Also the `max_len` will be determined by the `send` circuit cause it's that one that has 3 extra zeroes.